tests/specs/mcd/verification.k

```diff
@@ -467,5 +496,6 @@ module LEMMAS-MCD-COMMON
     rule M:Map [ K1 <- _ ] [ K2 <- V2 ] [ K3 <- V3 ] [ K1 <- V4 ] => M [ K1 <- V4 ] [ K2 <- V2 ] [ K3 <- V3 ] requires K1 =/=Int K2 andBool K1 =/=Int K3 [simplification]
 
     rule chop(X *Int (Y +Int pow256)) => chop(X *Int Y) [simplification]
+    rule chop(X *Int (Y +Int pow256)) => chop(X *Int Y) [simplification]
```

Comment:
This seems like a duplicate of the line above. Is it needed for something?

Reply:
Ah, no no, that was accidental, I didn't spot it, feel free to get rid of it.